package.json

```diff
@@ -61,12 +59,12 @@
     "extract-css-chunks-webpack-plugin": "^2.0.18",
     "extract-hoc": "^0.0.5",
     "extract-hoc-compose": "^0.0.1",
-    "extract-text-webpack-plugin": "^3.0.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
I believe this is no longer being supported or contributed to. See https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/749. 

You may want to follow the pattern in this PR which uses mini-css-extract-plugin instead: https://github.com/phoenixframework/phoenix/pull/2831/files